### PR TITLE
Replace plugin manager's 'Check now' button with 'jenkins-button'

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed replaces the "Check now" button on the update center with a jenkins button.

Before:

Screenshot 2022-09-28 at 22 48 51

After:

Screenshot 2022-09-28 at 22 48 01

### Proposed changelog entries

- Modernize update center check button.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7183"></a>

### CODE REVIEW

**Review by timja — COMMENTED** [review]
probably want to check in ATH to be safe

**Comment by NotMyFault** [comment]
> probably want to check in ATH to be safe

ATH is green.

**Review by timja — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]
(Untested.)

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: core/src/main/resources/hudson/PluginManager/check.jelly
Comment (uhafner) [inline core/src/main/resources/hudson/PluginManager/check.jelly:28]: I think we should have only one primary button on a page.:

```suggestion
  <f:link href="checkUpdatesServer" post="true" clazz="jenkins-button submit-button">
```

**Review by uhafner — CHANGES_REQUESTED** [review]

File: core/src/main/resources/hudson/PluginManager/check.jelly
Comment (NotMyFault) [reply core/src/main/resources/hudson/PluginManager/check.jelly:28]: > only one primary button on a page

Is there are a particular reason, why the blue button should be turned into a less intuitive and harder to distinguish, default button?

Screenshot 2022-09-29 at 23 51 38

Comment (uhafner) [reply core/src/main/resources/hudson/PluginManager/check.jelly:28]: If you think that default button has a bad design then we need to improve that in a different PR. My comment is only about the semantics of your button choice. 

An important UI design rule is that a page has a single primary action only (example: https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150). On the updates and install pages this should be the "Install" action. The check for updates button certainly is only a secondary action, hence I would use the associated class for this action.

Comment (NotMyFault) [reply core/src/main/resources/hudson/PluginManager/check.jelly:28]: > Which one is primary?

Good point. I was going off the default view and had only a couple of updates available, but I do understand the problem with having several primary buttons, that not really are meant to be primary.

Thanks for the feedback & clarification! :)

**Review by uhafner — APPROVED** [review]

**Pull request merged**
